src/services/customer.service.js

```diff
@@ -0,0 +1,21 @@
+import { Company } from "../models/company.model.js"
+
+const FetchCompanyById = async ( companyId ) => Company.findOne({companyId});
+
+const CreateCompany = async ( newRegistration ) => Company.create({...newRegistration});
+
+const FetchByToken = async ( verificationToken ) => Company.findOne({verificationToken});
+
+const UpdateVerificationStatus = async ( companyId ) => {
+    Company.findByIdAndUpdate({_id: companyId}, {isVerified : true}, {new: true})
```

Comment:
Is the $set parameter required for that route @Ant-Ghost or @singhayushh ? Or its something for older MongoDB ?